app/views/status_messages/bookmarklet.html.haml

```diff
@@ -1,9 +1,3 @@
--#   Copyright (c) 2010-2012, Diaspora Inc.  This file is
--#   licensed under the Affero General Public License version 3 or later.  See
--#   the COPYRIGHT file.
-
-= javascript_include_tag :home
```

Comment:
`fileuploader-custom` was before in that file which we already load in `main.js`.